py/desispec/fiberflat.py

```diff
@@ -475,7 +475,7 @@ def __init__(self, wave, fiberflat, ivar, mask=None, meanspec=None,
         self.wave = wave
         self.fiberflat = fiberflat
         self.ivar = ivar
-        self.mask = mask
+        self.mask = mask.astype(np.uint32)
```

Comment:
What insurance do you have that the input value of mask will always be a type that can be cast to `np.uint32`?  It says 32 bit in the documentation, but if someone puts in something 64-bit anyway, numpy will throw a (possibly obsure) exception.